development/getting_code_on_the_robot.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: Getting Code on the Robot
+---
+
+Getting Code on the Robot
+======================
+
+Getting your code on to the robot is quite simple.
+You will need to put your code on a USB drive. Which will need to be formatted with either FAT32 or exFAT.
+
+<details>
```

Comment:
These don't render properly at all

![image](https://user-images.githubusercontent.com/6527489/140644246-4993d89d-1263-463d-8a7a-027a28b87543.png)

Any reason not to just use regular `h2`s?